game/scripts/npc/abilities/lycan_shapeshift.txt

```diff
@@ -16,7 +16,7 @@
     "MaxLevel"                                            "5"
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "130 105 80 80 80"
+    "AbilityCooldown"                                     "80 70 60 60 60" //OAA
```

Comment:
With the bonus duration talent lowering the cd seems pretty dumb. I'd suggest just buffing his level durations to like 18/21/24/27/30 and maybe just buffing level 1 cd to like 110 level 2 to 100 and keeping the last 3 levels at 80 so like 110/100/80/80/80

Reply:
First level should be 100 seconds. Last level should be 60 seconds. I proposed ''100 90 80 70 60'' scaling because that doesn't mess up with power spike timings and its still a buff.